Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/usr/local/lib/python3.11/dist-packages/nbclient/client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/usr/local/lib/python3.11/dist-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/local/lib/python3.11/dist-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Analisis Histogram Petal Length by Class
plt.figure(figsize=(8, 5))
sns.histplot(data=result_df, x='petal length', hue='class', multiple='stack', palette='Blues', bins=10, edgecolor='black')
plt.title('Petal Length Histogram by Class')
plt.xlabel('Petal Length')
plt.ylabel('Count')
plt.show()
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m<ipython-input-9-ac3c1b9b52f0>[0m in [0;36m<cell line: 0>[0;34m()[0m
[1;32m      1[0m [0;31m# Analisis Histogram Petal Length by Class[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m [0mplt[0m[0;34m.[0m[0mfigure[0m[0;34m([0m[0mfigsize[0m[0;34m=[0m[0;34m([0m[0;36m8[0m[0;34m,[0m [0;36m5[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      3[0m [0msns[0m[0;34m.[0m[0mhistplot[0m[0;34m([0m[0mdata[0m[0;34m=[0m[0mresult_df[0m[0;34m,[0m [0mx[0m[0;34m=[0m[0;34m'petal length'[0m[0;34m,[0m [0mhue[0m[0;34m=[0m[0;34m'class'[0m[0;34m,[0m [0mmultiple[0m[0;34m=[0m[0;34m'stack'[0m[0;34m,[0m [0mpalette[0m[0;34m=[0m[0;34m'Blues'[0m[0;34m,[0m [0mbins[0m[0;34m=[0m[0;36m10[0m[0;34m,[0m [0medgecolor[0m[0;34m=[0m[0;34m'black'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m [0mplt[0m[0;34m.[0m[0mtitle[0m[0;34m([0m[0;34m'Petal Length Histogram by Class'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m [0mplt[0m[0;34m.[0m[0mxlabel[0m[0;34m([0m[0;34m'Petal Length'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mNameError[0m: name 'plt' is not defined

